api/symptom.go

```diff
@@ -151,6 +152,14 @@ func (s *Server) reportSymptoms(c *gin.Context) {
 				sentry.CaptureException(err)
 			}
 		}()
+
+		go func() {
```

Comment:
It isn't utils.TriggerAccountSymptomFollowUpNudge synchronized ? Why do we need a goroutine ?

Reply:
Just not to block this API. Same to `TriggerPOIUpdate` above.

Reply:
Should other triggers add a goroutine?  The last discussion is that behavior/symptom triggers are synchronized so ... is these two asynchronized?

Reply:
I do not know clearly about the definition of asynchronized you mentioned here. The job is invoked in the background. The cadence api call to invoke the job happens in the `reportSymptoms` API.